### ISSUE TITLE
Uncaught TypeError: undefined is not a function

### ISSUE THREAD

**Issue by erenburakalic** [lee-dohm/auto-copyright#14]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.209.0
**System**: Mac OS X 10.10.3
**Thrown From**: [auto-copyright](https://github.com/lee-dohm/auto-copyright) package, v0.4.1

### Stack Trace

Uncaught TypeError: undefined is not a function

```
At /Users/erenburakalic/.atom/packages/auto-copyright/lib/auto-copyright.coffee:33

TypeError: undefined is not a function
  at AutoCopyright.insert (/Users/erenburakalic/.atom/packages/auto-copyright/lib/auto-copyright.coffee:33:29)
  at atom-workspace.atom.commands.add.auto-copyright:insert (/Users/erenburakalic/.atom/packages/auto-copyright/lib/auto-copyright.coffee:27:10)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:238:29)
  at CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:153:19)
  at EventEmitter.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:70:30)
  at emitOne (events.js:77:13)
  at EventEmitter.emit (events.js:166:7)

```

### Commands

```
     -2:58.6.0 command-palette:toggle (ul.list-inline.tab-bar.inset-panel)
     -2:55.3.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -2:54.5.0 core:move-up (atom-text-editor.editor.mini.is-focused)
     -2:52.2.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -2:52.1.0 auto-copyright:update (ul.list-inline.tab-bar.inset-panel)
  2x -2:42.6.0 auto-copyright:insert (atom-pane.pane.active)
     -2:23.4.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  2x -1:23.9.0 auto-copyright:insert (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "seti-ui",
      "atom-dark-syntax"
    ]
  },
  "auto-copyright": {}
}
```

### Installed Packages

```coffee
# User
auto-copyright, v0.4.1
color-picker, v2.0.4
minimap, v4.9.4
seti-ui, v0.7.1

# Dev
No dev packages
```

**Comment by majk-p** [comment]
Same issue with atom 0.210.0 using Linux Mint mint
```
  TypeError: undefined is not a function
    at AutoCopyright.insert (/home/majk/.atom/packages/auto-copyright/lib/auto-copyright.coffee:33:29)
    at atom-workspace.atom.commands.add.auto-copyright:insert (/home/majk/.atom/packages/auto-copyright/lib/auto-copyright.coffee:27:10)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/opt/atom/resources/app.asar/src/command-registry.js:238:29)
    at /opt/atom/resources/app.asar/src/command-registry.js:3:61
    at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/opt/atom/resources/app.asar/node_modules/command-palette/lib/command-palette-view.js:159:32)
    at CommandPaletteView.module.exports.SelectListView.confirmSelection (/opt/atom/resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
    at space-pen-div.atom.commands.add.core:confirm (/opt/atom/resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/opt/atom/resources/app.asar/src/command-registry.js:238:29)
    at /opt/atom/resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/opt/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/opt/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
    at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/opt/atom/resources/app.asar/src/window-event-handler.js:180:20)```

**Comment by lee-dohm** [comment]
This should be fixed in the latest version, v0.5.0. Please let me know if you run into any issues.
